tdms/src/magnetic_field.cpp

```diff
@@ -0,0 +1,16 @@
+#include "complex"
+#include "globals.h"
+#include "field.h"
+
+using namespace std;
+
+
+void MagneticField::add_to_angular_norm(double f, int n, int Nt, SimulationParameters &params) {
+  angular_norm += phasor_norm(f, n, params.omega_an, params.dt, Nt);
+}
+
+complex<double> MagneticField::phasor_norm(double f, int n, double omega, double dt, int Nt){
+  return f
+         * exp( fmod(omega*((double) n + 0.5)*dt, 2*dcpi) * I)
```

Comment:
this might be a question for @prmunro

Reply:
I don't think this is a problem with the time convention. The time convention means that to go from a frequency domain field Ex to a time domain field ex, the following equation applies: ex = real( Ex*exp(-i\omega t)), when going from time domain data to frequency domain data the opposite sign is used, since this is the inverse operation.

If it was a sign error,  the error metric would be much larger as this would return the complex conjugate of the required field quantity.